pkgs/by-name/ct/ctags-lsp/package.nix

```
@@ -0,0 +1,48 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  nix-update-script,
+  universal-ctags,
+  versionCheckHook,
+}:
+buildGoModule rec {
+  pname = "ctags-lsp";
+  version = "0.6.1";
+  vendorHash = null;
+
+  src = fetchFromGitHub {
+    owner = "netmute";
+    repo = "ctags-lsp";
+    tag = "v${version}";
+    hash = "sha256-wSccfhVp1PDn/gj46r8BNskEuBuRIx1wydYAW1PV4cg=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/ctags-lsp \
+      --prefix PATH : ${lib.makeBinPath [ universal-ctags ]}
```

To make it easier to override by the user's own `ctags`, I'd suggest:

```suggestion
      --suffix PATH : ${lib.makeBinPath [ universal-ctags ]}
```